include/sway/tree/container.h

```diff
@@ -70,9 +70,8 @@ struct sway_container {
 	enum sway_container_layout prev_layout;
 	enum sway_container_layout workspace_layout;
 
-	// TODO convert to layout coordinates
+	// in output-local coordinates
 	double x, y;
```

Comment:
change to `sx` and `sy` and you can remove the comment.

Reply:
Mhh, this isn't even correct since output containers have layout coordinates.